app/views/govuk_component/metadata.raw.html.erb

```diff
@@ -42,14 +46,16 @@
       </dd>
     <% end %>
     <% if other.present? %>
-      <% other.each do |title, definition| %>
+      <% other.each_with_index do |(title, definition), index| %>
         <%
           definition ||= []
           definition = Array(definition)
         %>
         <% if definition.any? %>
           <dt><%= title %>:</dt>
-          <dd><%= definition.to_sentence.html_safe %></dd>
+          <dd>
+            <%= render file: 'govuk_component/metadata_sentence.raw', locals: { items: definition, toggle_id: index } %>
```

Comment:
We had a look in Kibana and it seemed to have an expected amount of requests, we want to do a little bit more investigation when deploying but are otherwise happy.